src/app/components/quotes/quotes.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { QuoteService } from '../../services/quote.service';
+import { AlertService } from '../../services/alert.service';
+
+@Component({
+  selector: 'mymicds-quotes',
+  templateUrl: './quotes.component.html',
+  styleUrls: ['./quotes.component.scss']
+})
+export class QuotesComponent implements OnInit {
+	
+  constructor(private quote:QuoteService, private alertService:AlertService) { };
+
+  ngOnInit() {
+	  this.quote.getQuote().subscribe((result) => {
+		  document.getElementById("quote").innerHTML = result.quote;
```

Comment:
Could you maybe use class instance variables instead? Typically, you don't want to directly use DOM methods with Angular unless you absolutely have to. In the component HTML, you could do something like `{{ result ? result : 'Loading quote...' }}` [You can learn more about the Angular template syntax here.](https://angular.io/docs/ts/latest/guide/template-syntax.html#!#interpolation)